numba_dpex/core/kernel_interface/spirv_kernel.py

```diff
@@ -139,6 +139,20 @@ def compile(
         self._llvm_module = kernel.module.__str__()
         self._module_name = kernel.name
 
+        # Dump LLVM IR if DEBUG flag is set.
+        if config.DEBUG:
```

Comment:
Can we use a new flag `NUMBA_DPEX_DUMP_KERNEL_LLVM`? The `DEBUG` seems to turn on debug symbols in some places.

Reply:
Added. Could not find any place where documentation needs to be updated with the env var. If I have missed it, let me know.